cmake/scripts/expand_instantiations.cc

```diff
@@ -19,11 +19,10 @@
 // is given as arguments on the command line and contains entries of the
 // following form:
 // --------------------
-// REAL_SCALARS    := { double; float; long double }
+// REAL_SCALARS    := { double; float}
 // COMPLEX_SCALARS := { std::complex<double>;
-//                      std::complex<float>;
-//                      std::complex<long double> }
-// VECTORS := { Vector<double>; Vector<float>; Vector<long double> }
+//                      std::complex<float>}
+// VECTORS := { Vector<double>; Vector<float>}
 // --------------------
 //
 // The input file is typically located in share/deal.II/template-arguments in
```

Comment:
While you are here, would you mind also fixing this path?

Reply:
This file has long since been moved to `cmake/config/template-arguments.in`.

Reply:
Done!

Reply:
Sorry, I put my comment on the wrong line before: the previous comment was correct (the file really is at `share/deal.II/template-arguments` in the build directory) but the next clause should be `and it is built from cmake/config/template-arguments.in`.